tools/bench_deepgemm_stream.py

```
@@ -0,0 +1,426 @@
+# ---------------------------------------------------------------------------
+try:
+    import torch.cuda.nvtx as nvtx
+    _HAS_NVTX = True
+except Exception:
+    _HAS_NVTX = False
+
+_PROFILE = False
+
+
+@contextmanager
+def nvtx_range(name, color=None):
```

Ruff (0.15.1)

[warning] Unused function argument: `color`

(ARG001)